.jenkins/pytorch/win-test-helpers/setup_pytorch_env.bat

```diff
@@ -36,7 +36,7 @@ popd
 =======
```

Comment:
Just a side note, I have been thinking about this "=======" line for as long as I remember.  I wonder what it even means here, but the bat script works so I have no incentive to remove it :P